SPECS-EXTENDED/perl-Sys-Virt/perl-Sys-Virt.spec

```diff
@@ -46,7 +46,7 @@ machine management APIs. This allows machines running within arbitrary
 virtualization containers to be managed with a consistent API.
 
 %prep
-%autosetup -S git -n Sys-Virt-v%{version}
+%autosetup -n Sys-Virt-v%{version}
```

Comment:
Did you verify that it works with the check build as well? Does it have to?

Reply:
Ah, including the git BR with the check dependencies was a mistake. Removed that BR entirely.